CDS/scripts/dailymotion_upload.pl

```diff
@@ -188,45 +188,46 @@
 }
 
 
+sub is_imageurl_valid {
```

Comment:
I'd normally put subroutine definitions at the top of the script, just below the imports, then a comment marking #START MAIN when the script's main func starts

Reply:
I'd also just return a boolean value, 1 or 0,

Reply:
Done.